xml/ha_storage_protection.xml

```diff
@@ -781,18 +780,44 @@ Timeout (msgwait)  : 10
        would be appropriate if the <literal>msgwait</literal> timeout value for
        SBD was set to <literal>30</literal> seconds.</para>
      </callout>
-     <callout arearefs="co.ha.sbd.st.rsc">
-      <para>
-       The &stonith; resource to use for SBD.</para>
-     </callout>
-     <callout arearefs="co.ha.sbd.pm_delay_max">
+   </calloutlist>
+  </step>
+  <step>
+   <para>For a two-node cluster, decide if you want predictable or random delays. For
+   other cluster setups you do not need to set this parameter.</para>
+   <variablelist>
+    <varlistentry>
+     <term>Predictable Static Delays</term>
+     <listitem>
+      <para>This parameter enables a static delay before executing &stonith; actions.
+      It ensures that the nodes do not fence each other if separate fencing resources
+      and different delay values are being used. The targeted node will
+      loose in a <quote>fencing race</quote>.
+      The parameter can be used to <quote>mark</quote> a specific node to survive
+      in case of a split-brain scenario in a two-node cluster.
+      To make this succeed, it is essential to create two primitive &stonith;
+      devices for each node. In the following configuration, &node1; will win
+      and survive in case of a split-brain scenario:
+      </para>
+      <screen>&prompt.crm.conf;<command>primitive</command> st-sbd-&node1; stonith:external/sbd params \
+       pcmk_host_list=&node1; pcmk_delay_base=20
+&prompt.crm.conf;<command>primitive</command> st-sbd-&node2; stonith:external/sbd params \
+       pcmk_host_list=&node2; pcmk_delay_base=0</screen>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term>Dynamic Random Delays</term>
+     <listitem>
       <para>This parameter prevents double fencing when using slow devices such
        as SBD. It adds a random delay for &stonith; actions on the fencing device.
        It is especially important for two-node clusters where otherwise both nodes
        might try to fence each other in case of a split brain scenario.</para>
```

Comment:
Given the new context, how about something like:

Rather than a static delay targeting a specific node, the parameter `pcmk_delay_max` adds a random delay for any fencing with the fencing resource to prevent double reset. Unlike `pcmk_delay_base`, this parameter can be specified for an unified fencing resource targeting multiple nodes.

Reply:
@tomschr, given that https://github.com/SUSE/doc-sleha/pull/168 is merged, also "Predictable static delays" introduced earlier, we now have the kind of new context here.  What do you think about the above comment? :-)

Reply:
@gao-yan As this branch is already merged, I've created a new PR with your comments. I've tried to tackle all of your comments. Thanks for all your help, much appreciated! :100: